### ISSUE TITLE
not working on new firmware?

### ISSUE THREAD

**Issue by SpudGunMan** [NStrijbosch/hub2hub#6]
I cant load this project on spike or Mindstorms also the installer file is "only spike" will also need to provide the python direct or a LMS file as well for Mindstorms RI4 brick

Traceback (most recent call last):  File "programrunner/__init__.py", line 1, in start_programMemoryError: memory allocation failed, allocating 14177 bytes

install_version = '0.1.1'

spike 4.0.0.5 firmware

**Comment by NStrijbosch** [comment]
That is correct. The current project is only successfully tested with the SPIKE Prime apps 1.3.5, 1.3.4 and 1.3.3. The newest versions of the apps and corresponding firmware has an updated version of the `ubluetooth` module that breaks the current project. 

This and also the memory issues you mention above, I do not have plans to support newer versions. Instead I am hopeful that [pybricks](pybricks.com) will provide communication between all PoweredUp hubs in the near feature.

**Comment by haylinmoore** [comment]
That is very disappointing, so at the moment there seems to be no current library for bluetooth for Spike?
